R/utilities-units.R

```diff
@@ -77,9 +95,9 @@ getBaseUnit <- function(dimension) {
 #' valuesInBaseUnit <- toBaseUnit(par, c(1000, 2000, 3000), "ml")
 #' @export
 toBaseUnit <- function(quantityOrDimension, values, unit, molWeight = NULL, molWeightUnit = NULL) {
-  validateIsOfType(quantityOrDimension, c(Quantity, "character"))
-  validateIsNumeric(values, nullAllowed = TRUE)
-  validateIsNumeric(molWeight, nullAllowed = TRUE)
+  ospsuite.utils::validateIsOfType(quantityOrDimension, c(Quantity, "character"))
```

Comment:
Should this instead be `c("Quantity", "character")`?

Reply:
Using string names for all `isOfType` and `validateIsOfType` now.